### ISSUE TITLE
Change label for Show Values checkbox

### ISSUE THREAD

**Issue by terracoda** [phetsims/molarity#114]
@twant, @Matthew-phet, @emily-phet and I think that the label for the "Show Values" checkbox should be changed to "Solution Values". 

This change should be made to both the visual sim and the accessible PDOM.

No changes to help text or alerts for this checkbox, just the label change.

**Comment by terracoda** [comment]
@Matthew-phet, this change looks and sounds really good to me.
Thanks @twant! Verified and closing.
